adventure_game/main.py

```diff
@@ -38,10 +38,10 @@ def intro():
     greet_user('Ellen')
     greet_user('Sophie')
     greet_user('Joe')
+
     # TO DO: Ellen - figure out how to print items as a string without []
-    # Shows the type of items below
-    print(type(items))
-    print_pause(f"You find yourself in a dark cave and you only have a {items} in your hand")
+    string_items = ', '.join(items)
+    print_pause("You find yourself in a dark cave and you only have a " + string_items + " in your hand")
```

Comment:
Good Job, there's a couple different ways you could have also added the string_items variable to the phrase: 
1. f string 
2. { } - using these brackets